iaso/api/entity.py

```diff
@@ -56,7 +56,7 @@ class Meta:
 
     def get_attributes(self, entity: Entity):
         if entity.attributes:
-            return entity.attributes.as_full_model()
+            return entity.attributes.as_dict()
```

Comment:
You're sure that this does not break the entity detail screen?

Reply:
I've tested and reviewed the code and I can't find any reason to use the `as_full_model()` here